package/chocolatey/README.md

```diff
@@ -0,0 +1,8 @@
+Chocolatey Punktf package
+===
+
+## Release a new version
+
+1. Change url and checksum in `tools/chocolateyinstall.ps1`
+2. Adjust version number in `punktf.nuget`
```

Comment:
Should `punktf.nuget` be renamed to `punktf.nuspec` ?

Reply:
Yep, thx!